com.openterface.openterfaceQT.yaml

```
@@ -0,0 +1,37 @@
+app-id: com.openterface.openterfaceQT
+runtime: org.kde.Platform
+runtime-version: "6.9"
+sdk: org.kde.Sdk
+command: openterfaceQT
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=all
+  - --filesystem=/run/udev:ro
+
+modules:
+  - shared-modules/libusb/libusb.json
+
+  - name: openterfaceQT
+    buildsystem: qmake
+    config-opts:
+      - "LIBS+=-L/app/lib -lusb-1.0"
+      - "INCLUDEPATH+=/app/include/libusb-1.0"
+    sources:
+      - type: git
+        url: https://github.com/TechxArtisanStudio/Openterface_QT.git
+        tag: 0.3.8
+
+    build-options:
+      env:
+        QT_SELECT: qt6
+    build-commands:
+      - install -Dm755 openterfaceQT /app/bin/openterfaceQT
+      - test -d config && cp -r config /app/bin/ || echo "No config directory found"
+    post-install:
+      - install -Dm644 com.openterface.openterfaceQT.desktop /app/share/applications/com.openterface.openterfaceQT.desktop
+      - install -Dm644 com.openterface.openterfaceQT.desktop /app/export/share/applications/com.openterface.openterfaceQT.desktop
```

```suggestion
      - install -Dm644 com.openterface.openterfaceQT.desktop /app/export/share/applications/com.openterface.openterfaceQT.desktop
```

you NEVER copy anything there. I don't know where in the documentation it say to do so but it is incorrect.